doc/install/install-tf.2.3.md

```diff
@@ -1,16 +1,16 @@
 # Install TensorFlow's C++ interface 
-The tensorflow's C++ interface will be compiled from the source code. Firstly one installs bazel. The bazel version 3.1.0 should be used. A full instruction of bazel installation can be found [here](https://docs.bazel.build/versions/master/install.html).
+The tensorflow's C++ interface will be compiled from the source code. Firstly one installs bazel. [bazelisk](https://github.com/bazelbuild/bazelisk) can be lanuched to use [bazel](https://github.com/bazelbuild/bazel).
+
 ```bash
-cd /some/workspace
-wget https://github.com/bazelbuild/bazel/releases/download/3.1.0/bazel-3.1.0-installer-linux-x86_64.sh
-chmod +x bazel-3.1.0-installer-linux-x86_64.sh
+wget https://github.com/bazelbuild/bazelisk/releases/download/v1.11.0/bazelisk-linux-amd64 -O /some/workspace/bazel/bin/bazel
+chmod +x /some/workspace/bazel/bin/bazel
 ./bazel-3.1.0-installer-linux-x86_64.sh --prefix /some/workspace/bazel
 export PATH=/some/workspace/bazel/bin:$PATH
 ```
 
 Firstly get the source code of the TensorFlow
 ```bash
-git clone https://github.com/tensorflow/tensorflow tensorflow -b v2.3.0 --depth=1
+git clone https://github.com/tensorflow/tensorflow tensorflow -b v2.8.0 --depth=1
```

Comment:
The mane of the file is "tf.2.3.md",  shall we keep it as is and add a new doc for tf 2.8?

Reply:
done